src/__tests__/get-by-errors.js

```diff
@@ -97,6 +97,33 @@ cases(
   },
 )
 
+test.each([['getByText'], ['getByLabelText']])(
+  '%s query will print the playground link when enabled in the config',
+  query => {
+    configure({printPlaygroundLink: true})
+    document.body.innerHTML = '<div>Hello</div>'
+    expect(() => screen[query]('TEST QUERY')).toThrow(/playground/i)
```

Comment:
Could you include a more sensible partial match here? It's not clear that `playground` refers to the link. This would still pass if we mention the word `playground` but not include the link, no?

Reply:
Good point! It seems like a great use-case for `toThrowErrorMatchingSnapshot`, so I've changed it to that instead :-)